W-13932478 retest main tuts duke

dukesphere opened the pull request.

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
Merge branch 'open-beta-sept' into W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
W-13932478-retest-main-tuts-duke

dukesphere committed:
Merge branch 'open-beta-sept' into W-13932478-retest-main-tuts-duke

chris-campbell9 commented on modules/ROOT/pages/tut-design-api-specification-from-scratch.adoc:
```diff
@@ -27,25 +27,49 @@ Start creating the American Flights API specification.
+. Proceed to <<open-project>>.
+
+[[open-project]]
+== Open the Workspace for Your API Specification Project
```
These next four steps are unnecessary on Desktop, because the project's workspace is automatically opened after selecting "create project". Does this behavior differ on web? @dukesphere

dukesphere replied:
oh good! I used to get an issue where if I created a folder to put the project in from the *Project Location* field, I had to navigate to the project folder. Maybe it was fixed or ¯\_(ツ)_/¯ . Removing the section. Thx!

dukesphere replied:
@chris-campbell9 Confirmed and removed section. Thx 😄

dukesphere committed:
Update tut-design-api-specification-from-scratch.adoc

removing step on opening the workspace folder; not needed as reported by Chris; tested and confirmed

chris-campbell9 commented on modules/ROOT/pages/tut-design-api-specification-from-scratch.adoc:
```diff
@@ -71,6 +77,8 @@ title: American Flights API
 +
 image::acb-api-console.png["A specification with get and post methods"]
 +
+Wait for the API Console to load. 
```
On Desktop, I had to click back in the editor window to get the API Console to display my API. If I didn't click in the editor, the API Console was present but empty, which was confusing. Does this behavior differ from web? Or perhaps it's a product bug? I can pursue this with eng, but I thought I'd check to see if you knew offhand. @dukesphere

dukesphere replied:
Ohhh, that happens to me sometimes but I didn't connect the dots. Seems like a bug to me, I/we can tell them to click in the edit window for now. Thx!

dukesphere replied:
I thought it was a loading delay or something.

chris-campbell9 commented on modules/ROOT/pages/tut-design-api-specification-from-scratch.adoc:
```diff
@@ -108,4 +116,4 @@ title: American Flights API
 . Open the API Console to display the nested resource:
```
```suggestion
. View the nested resource in the API Console:
```
The API Console was already open due to following the steps in the previous section. Not a big deal, but I'm raising a question because I wonder if the customer could close the console between sections? If so, then disregard my suggestion. @dukesphere

dukesphere committed:
Update modules/ROOT/pages/tut-design-api-specification-from-scratch.adoc

Co-authored-by: chris-campbell9 <75496572+chris-campbell9@users.noreply.github.com>

dukesphere commented on modules/ROOT/pages/tut-design-api-specification-from-scratch.adoc:
```diff
@@ -71,6 +77,8 @@ title: American Flights API
 +
 image::acb-api-console.png["A specification with get and post methods"]
 +
+Wait for the API Console to load. 
```
```suggestion
Click in the RAML file you are editing, and wait for the API Console to load. 
```

dukesphere committed:
Update tut-design-api-specification-from-scratch.adoc

dukesphere committed:
Merge branch 'open-beta-sept' into W-13932478-retest-main-tuts-duke
